Engine/PSScriptAnalyzer.psm1

```diff
@@ -20,9 +20,13 @@ if (($PSVersionTable.Keys -contains "PSEdition") -and ($PSVersionTable.PSEdition
 else {
     if ($PSVersionTable.PSVersion.Major -eq 3) {
         $binaryModuleRoot = Join-Path -Path $PSModuleRoot -ChildPath 'PSv3'
+        # Without this, PSSA tries to load this from $PSHome
+        Add-Type -Path "$binaryModuleRoot/Newtonsoft.Json.dll"
     }
     elseif ($PSVersionTable.PSVersion.Major -eq 4) {
```

Comment:
We should avoid this duplication of the v3 if statement above and just merge them into one if statement for 3 and 4 where express the difference (PSv3 vs PSv4 folder names) as an interpolated string: `"PSv$($$PSVersionTable.PSVersion.Major)"`